scrapers/AEBN.yml

```diff
@@ -10,6 +15,53 @@ movieByURL:
       - aebn.com
     scraper: movieScraper
 xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //div[@class='dts-section-page-heading-title']/h1
+      Gender:
+        selector: //li[@class="section-detail-list-item-gender"]/text()
+        postProcess:
+          - map:
+              TS: transgender_female
```

Comment:
While this is not always correct, AEBN only has 3 gender categories Female, Male, and TS. And since most Trans content is from Transgender Females, I feel like it's better than leaving it unamtched.